### DISCUSSION TITLE
Unclear labels in the Project page

### DISCUSSION THREAD

**Discussion by johardi** [redivis/meta#170] [Feature requests]
**What is the problem you'd like us to address**
I found some labels on the project page are unclear.
1. "Lists" in the top left-hand side doesn't say clearly it is a value list
2. "Search project" doesn't mean searching for a specific project, but instead, it searches nodes in this project.

**Describe a solution (if you have one in mind)**
1. Remove from the top menu and add the option to create a new value list in the "Row filters" drop-down value selector.
2. "Search project nodes"?

**Describe other related products we could check out (if applicable)**
N/A

**Any additional context**
N/A

**Comment by imathews** [comment]
@johardi we've made some substantial UI updates to the project tool in our [latest release](https://github.com/redivis/meta/releases/tag/0.20.0) that should reduce confusion in navigating the project.
